src/utils/SortingNetworkGenerator.java

```diff
@@ -99,6 +107,14 @@ public static String encodeNetworkAndDisplay(String name, Integer[] indices, int
             indicesInt[i] = indices[i];
         }
         if (!encodeNetwork(indicesInt, path)) {
+			if (indicesInt.length == 0) {
+				JOptionPane.showMessageDialog(null, "Sort does not compare indices; An empty sorting network cannot be generated.",
+					"File not saved", JOptionPane.ERROR_MESSAGE);
+			}
+			else {
+				JOptionPane.showMessageDialog(null, "Sorting network too large. (Exceeds " + formatter.format(LIMIT) + " comparators)",
+					"File not saved", JOptionPane.ERROR_MESSAGE);
+			}
```

Comment:
Tabs here (and at lines 26 and 127), but this file uses spaces

Reply:
Fixed!